ppa/archive/templates/archive/snippets/search_form.html

```diff
@@ -1,6 +1,7 @@
 {% load semanticui humanize %}
 <form class="ui form{% if error %} error{% endif %}">
     <fieldset class="ui vertical stackable grid segment">
+        <legend class="sr-only">text search fields</legend>
```

Comment:
I wonder if "keyword" would make more sense than "text" ?

Reply:
i feel like some people think "keyword" is necessarily different from field scoped things like "title", "author" even though it's not. "text" is kinda vague though since it could mean "textual" (i.e. non-numeric) or the actual text of the pages in the PPA ("full text")....

@bwhicks do you have thoughts?

Reply:
sounds fair to me, thanks for explaning